assemblies/rosa-private-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="private-cluster"]
+:context: private-cluster
+
+= Configuring a private cluster
+
+[role="_abstract"]
+An {product-title} cluster can be made private so that internal applications can be hosted inside a corporate network. In addition, private clusters can be configured to have only internal API endpoints for increased security.
+
+// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager* (OCM).
+
+Privacy settings can be configured during cluster creation or after a cluster is established.
+
+// [NOTE]
```

Comment:
Is your intention to comment out the admonition paragraph, too? Without a preview build, I can't tell if it is sufficient to comment out just `[NOTE]` here, so just wanted to point it out.

Reply:
AFAIK, you'd want to use "ElasticSearch load balancer" (lowercase) as I don't think ES has a productized load balancer.

Reply:
To comment out the whole chunk, you need to use:
```
////
[NOTE]
====
Red Hat Service Reliability Engineers (SREs) can access a public or private cluster through the `cloud-ingress-operator` and existing ElasticSearch Load Balancer or Amazon S3 framework. SREs can access clusters through a secure endpoint to perform maintenance and service tasks.
====
////
```

Reply:
Thank you, I commented out the entire chunk for now.